sdk/python/tests/utils/e2e_test_validation.py

```diff
@@ -164,8 +164,15 @@ def _check_offline_and_online_features(
                 )
 
 
-def make_feature_store_yaml(project, test_repo_config, repo_dir_name: Path):
-    offline_creator: DataSourceCreator = test_repo_config.offline_store_creator(project)
+def make_feature_store_yaml(
+    project,
+    test_repo_config,
+    repo_dir_name: Path,
+    fixture_request: Optional[pytest.FixtureRequest],
```

Comment:
you shouldn't need to pass a fixture_request actually. the test environment already has a datasourcecreator